web/profiles/joinup/joinup.links.menu.yml

```diff
@@ -1,3 +1,17 @@
+joinup.admin_reporting:
+  route_name: joinup.admin_reporting
+  parent: system.admin
+  description: 'Access Joinup reports.'
+  title: 'Reporting'
+  weight: 75
+
+joinup.compatibility_documents:
+  route_name: entity.compatibility_document.collection
+  parent: system.admin
+  description: 'Overview of licence compatibility documents.'
+  title: 'Compatibility documents'
+  weight: 60
```

Comment:
I'm questioning the location of this link. IMHO, it should live in the `joinup_licence` module.